public/index.html

```diff
@@ -1,19 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width,initial-scale=1.0">
```

Comment:
Can you confirm if removing this tag the responsive continue working ?

Reply:
I just confirmen please don't remove this tags also this tag `<meta charset="utf-8">` to allow the browser knows what charset conversion we are using.

Reply:
This was removed by mistake.